domain/domain.go

```diff
@@ -744,12 +743,11 @@ func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.R
 		return sysFactory(do)
 	}
 	sysCtxPool := pools.NewResourcePool(sysFac, 2, 2, resourceIdleTimeout)
-	ctx, cancelFunc := context.WithCancel(context.Background())
-	do.cancel = cancelFunc
+	do.Context, do.cancel = context.WithCancel(context.Background())
 	callback := &ddlCallback{do: do}
 	d := do.ddl
 	do.ddl = ddl.NewDDL(
-		ctx,
+		do,
```

Comment:
Could we use `do.Context` instead of `do`? I think it more clear to know the `NewDDL`'s argument. If we can update it please update all of the similar places.

Reply:
Here it is.